src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -40,7 +40,7 @@ def _get_client(service, parsed_args):
     account_name = parsed_args.account_name or os.getenv('AZURE_STORAGE_ACCOUNT')
     account_key = parsed_args.account_key or os.getenv('AZURE_STORAGE_KEY')
     connection_string = parsed_args.connection_string or os.getenv('AZURE_STORAGE_CONNECTION_STRING')
-    sas_token = parsed_args.sas_token or os.getenv('AZURE_SAS_TOKEN')
+    sas_token = parsed_args.sas_token or os.getenv('AZURE_STORAGE_SAS_TOKEN')
```

Comment:
>os.getenv('AZURE_STORAGE_SAS_TOKEN') [](start = 41, length = 36)

I thought Derek had added generic support for environment variable-backed params.  I couldn't find any code/PRs for it though.  Probably worth checking with him